packages/react-component-library/src/enhancers/index.tsx

```diff
@@ -0,0 +1,19 @@
+import TextInput from '../components/TextInput'
+import Checkbox from '../components/Checkbox'
+import Radio from '../components/Radio'
+
+import withFormik from './withFormik'
+
+const Formik: object = {
```

Comment:
import { Checkbox, Radio, TextInput } from '../components'

const Formik = [Checkbox, Radio, TextInput].map(component => withFormik(component))

Reply:
Changed import statements but want Formik to be an object so not using map